ext/json/tests/pass001.phpt

```diff
@@ -464,7 +464,7 @@ array(14) {
     float(1.23456789E-13)
     ["E"]=>
     float(1.23456789E+34)
-    ["_empty_"]=>
+    [""]=>
```

Comment:
Not really sure which one do you mean as github is showing this comment on two different places (github is being wierd though :) ). One in the commit view (File changed)
```
> -    ["_empty_"]=>
> +    [""]=>
```
which is to replace `_empty_` and one in Conservation which shows
```
 -    ["e"]=>
+    ["e"]=>
```
which I'm not really sure. It might be some new line but if I look from the command line `git diff`, I don't see anything so might also be some github diff issue.

Anyway there isn't anything special that we should worry about :) It's just about not using `_empty_` and make decoding array and object a bit more consistent!